caddy/Caddyfile

```diff
@@ -17,104 +17,116 @@ jwt {
 
 
 jwt {
-    path /model-documentation/2019/test-group
-    allow test-group true
+    path /model-documentation/2019/HPV-Harvard-Kim
+    allow HPV true
     publickey /public_key.pem
 }
 
 jwt {
-    path /model-documentation/2019/Cambridge-Trotter
-    allow Cambridge-Trotter true
+    path /model-documentation/2019/HPV-LSHTM-Jit
+    allow HPV true
     publickey /public_key.pem
 }
 
 jwt {
-    path /model-documentation/2019/CDA-Razavi
-    allow CDA-Razavi true
+    path /model-documentation/2019/HepB-CDA-Razavi
+    allow HepB true
     publickey /public_key.pem
 }
 
 jwt {
-    path /model-documentation/2019/JHU-Lessler
-    allow JHU-Lessler true
+    path /model-documentation/2019/HepB-IC-Hallett
+    allow HepB true
     publickey /public_key.pem
 }
 
 jwt {
-    path /model-documentation/2019/LSHTM-Clark
-    allow LSHTM-Clark true
+    path /model-documentation/2019/HepB-Li
+    allow HepB true
     publickey /public_key.pem
 }
 
 jwt {
-    path /model-documentation/2019/LSHTM-Jit
-    allow LSHTM-Jit true
+    path /model-documentation/2019/HibPCV-JHU-Tam
+    allow HibPCV true
```

Comment:
This claim won't show up in the token because it's not a disease id. The token will have multiple claims, one for each disease id - can we reflect that with one allow rule per disease? Otherwise just `allow Hib true` would be ok in this instance.

Reply:
So ideally this would be

```
allow Hib true
allow PCV true
```
?

Reply:
if that's valid caddy syntax, yes

Reply:
actually I guess not

Reply:
yeah, with caddy it's an OR only

Reply:
It was lots easier to go with Hib only